core/src/Kokkos_View.hpp

```diff
@@ -598,28 +627,33 @@ class View : public ViewTraits<DataType, Properties...> {
   /** \brief  Compatible view of array of scalar types */
   using array_type =
       View<typename traits::scalar_array_type, typename traits::array_layout,
-           typename traits::device_type, typename traits::memory_traits>;
+           typename traits::device_type, typename traits::hooks_policy,
```

Comment:
I added these arguments because I realized it would be an oversight not to -- would like additional feedback on this (it already caused me to have to adjust the python interop static tests). Would this possibly break any applications? @dalg24

Reply:
it might break external stuff, but I doubt there is anything we can do about that if we want the hook thing in there.